buildconfig/download_win_prebuilt.py

```
@@ -86,8 +86,8 @@ def get_urls(x86=True, x64=True):
         'e3c79c1bde7a4b2064d1cd660d357e82a7f7e856'
         ],
         [
-        'https://github.com/pygame-community/SDL_image/releases/download/2.8.10-pgce/SDL2_image-devel-2.8.10-VCpgce.zip',
-        'daf020b5112b2330bcf89f4335415bfd4e65ed03'
+        'https://github.com/pygame-community/SDL_image/releases/download/2.8.12-pgce/SDL2_image-devel-2.8.12-VCpgce.zip',
```

Flake8 (7.3.0)

[error] continuation line missing indentation or outdented

(E122)